src/solc_0.8/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -29,20 +29,22 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
     mapping(address => mapping(address => bool)) internal _operatorsForAll;
     mapping(uint256 => address) internal _operators;
 
-    constructor(address metaTransactionContract, address admin) {
-        _admin = admin;
-        _setMetaTransactionProcessor(metaTransactionContract, METATX_SANDBOX);
-    }
+    constructor(address trustedForwarder)
```

Comment:
I guess we still the admin for super operators ?

Reply:
yeah, and for `WithMinter` as well.